### PR TITLE
Update shredmod.is-an.app.json

### PR DESCRIPTION
**Pull request by familycomicsstudios**
### Pulling my new changes.

---

- [x] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/shredmod.is-an.app.json [hunk 1]
```diff
@@ -9,5 +9,24 @@
...
+      "record": {
+        "CNAME": "shredmod.github.io/extensions/"
+      }
+    },
+    {
+      "subdomain": "docs",
+      "record": {
+        "CNAME": "shredmod.github.io/extensions/"
+      }
+    }
```
Comment (tarampampam) [inline domains/shredmod.is-an.app.json:31]:
```suggestion
  },
  "nested": [
    {
      "subdomain": "packager",
      "record": {
        "CNAME": "shredmod.github.io"
      }
    },
    {
      "subdomain": "extensions",
      "record": {
        "CNAME": "shredmod.github.io"
      }
    },
    {
      "subdomain": "docs",
      "record": {
        "CNAME": "shredmod.github.io"
      }
    }
  ]
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/shredmod.is-an.app.json [hunk 2]
```diff
@@ -9,5 +9,25 @@
...
+      }
+    },
+    {
+      "subdomain": "extensions",
+      "record": {
+        "CNAME": "shredmod.github.io"
+      }
+    },
+    {
+      "subdomain": "docs",
```
Comment (tarampampam) [inline domains/shredmod.is-an.app.json:27]:
```suggestion
      "subdomain": "documentations",
```

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
Check now, please

**Comment by familycomicsstudios** [comment]
Forgot to pull.

**Comment by tarampampam** [comment]
Check the changes that I made

**Comment by familycomicsstudios** [comment]
> Check the changes that I made

You can merge this now!

**Pull request merged**